source/includes/_invoice.md

```diff
@@ -2144,9 +1236,6 @@ class Invoice {
    "auditLogs":[]
 }
```

Comment:
I think an empty body is sent, could you please confirm and update if required?

Reply:
Hi Reshma, for this one for example https://killbill.github.io/slate/index.html?ruby#invoice-adjust-an-invoice-item the ruby one showed the content in the response. I used that one to show the common response in place of leaving it empty.